### ISSUE TITLE
Could you provide demo?

### ISSUE THREAD

**Issue by ZZYuting** [kenziyuliu/MS-G3D#2]
Hi, your paper is amazing good, I'm interesting about your work. Could you provide some demo?  Thanks a lot!

**Comment by kenziyuliu** [comment]
Hi there, thanks for your interest! :) 

Right now I might not have time to code up a demo but I would encourage you to take a look at the [original ST-GCN repo](https://github.com/yysijie/st-gcn/blob/master/OLD_README.md) where there might be some handy visualization tools you could use. I might get to this later.

**Comment by ZZYuting** [comment]
> Hi there, thanks for your interest! :)
--------------------------------
update:
I public a more stable demo, including training your custom datasets using HRNet, employ the pytorch model to onnx model and real_time demo for both videos and webcam. The codes is avaiable here: https://github.com/ZZYuting/GCN_ActionRecongtionTools.git
Hoping guys to star~ Details Readme will be updated.

**Comment by kenziyuliu** [comment]
Hi there, 

In terms of performance, I think single-stream models should be good enough without ensembles (pretrained models give >35% Top-1, IIRC, which already outperform ST-GCN).

One possible reason could be that ST-GCN uses a different input normalization scheme compared to MS-G3D (which is based on 2s-AGCN). Also, I would suggest trying on other actions to see your issues are consistent :)

**Comment by ZZYuting** [comment]
> Hi there,
> 
> In terms of performance, I think single-stream models should be good enough without ensembles (pretrained models give >35% Top-1, IIRC, which already outperform ST-GCN).
> 
> One possible reason could be that ST-GCN uses a different input normalization scheme compared to MS-G3D (which is based on 2s-AGCN). Also, I would suggest trying on other actions to see your issues are consistent :)

Thanks a lot!

**Comment by ZZYuting** [comment]
I'm training from scratch using "python3 main.py --config ./config/kinetics-skeleton/train_joint.yaml --work-dir train_kinetics_joint_resume_paper --half --device 0 1 2 3",  after training 40 epoches, the acc in test seems only 15%, is something wrong with my config? 
that's my tensorboard loss and acc:
![img](https://s1.ax1x.com/2020/06/01/tGAsVe.png)
![img](https://s1.ax1x.com/2020/06/01/tGAOx0.png)

**Comment by kenziyuliu** [comment]
Hi there,

Several suggestions:
- Make sure you use 16 samples per GPU worker during forward pass (or batch norm may be affected), so with 4 GPUs you should do `--device 0 1 2 3 --batch-size 64 --forward-batch-size 64`; See README.md
- There was a learning rate decay schedule for training the model (paper section 4.2) and you should expect to see a sudden drop in performance like this if you resuming training with the default (large) learning rate

**Comment by ZZYuting** [comment]
> Hi there,
> 
> Several suggestions:
> 
> * Make sure you use 16 samples per GPU worker during forward pass (or batch norm may be affected), so with 4 GPUs you should do `--device 0 1 2 3 --batch-size 64 --forward-batch-size 64`; See README.md
> * There was a learning rate decay schedule for training the model (paper section 4.2) and you should expect to see a sudden drop in performance like this if you resuming training with the default (large) learning rate

It works well. Thanks~

**Comment by kenziyuliu** [comment]
Hi @ZZYuting, 

I have made a simple demo video here: https://youtu.be/5TcHIIece38  (credit to the authors of [this paper](https://arxiv.org/abs/2007.02072)).

I will close this issue for now :)

**Comment by pranavgundewar** [comment]
> > Hi there, thanks for your interest! :)
> 
> update:
> I public a more stable demo, including training your custom datasets using HRNet, employ the pytorch model to onnx model and real_time demo for both videos and webcam. The codes is avaiable here: https://github.com/ZZYuting/GCN_ActionRecongtionTools.git
> Hoping guys to star~ Details Readme will be updated.

I am unable to find the demo. Maybe you have moved the repo?

**Comment by kenziyuliu** [comment]
@pranavgundewar 

> I am unable to find the demo. Maybe you have moved the repo?

I haven't looked into it much, but I think [these guys](https://github.com/skelemoa/quovadis) made an open-source demo, and my demo video from README was courtesy of them.

**Comment by wwdok** [comment]
> > Hi there, thanks for your interest! :)
> 
> update:
> I public a more stable demo, including training your custom datasets using HRNet, employ the pytorch model to onnx model and real_time demo for both videos and webcam. The codes is avaiable here: https://github.com/ZZYuting/GCN_ActionRecongtionTools.git
> Hoping guys to star~ Details Readme will be updated.

Hi, the repo has been 404...
